addons/payment/static/src/js/payment_form_mixin.js

```diff
@@ -280,16 +280,16 @@ odoo.define('payment.payment_form_mixin', require => {
         /**
          * Prepare the params to send to the transaction route.
          *
-         * For an acquirer to overwrite generic params or to add acquirer-specific ones, it must
+         * For a provider to overwrite generic params or to add provider-specific ones, it must
          * override this method and return the extended transaction route params.
          *
          * @private
-         * @param {string} provider - The provider of the selected payment option's acquirer
+         * @param {string} code - The code of the selected payment option provider
          * @param {number} paymentOptionId - The id of the selected payment option
          * @param {string} flow - The online payment flow of the selected payment option
          * @return {object} The transaction route params
          */
-        _prepareTransactionRouteParams: function (provider, paymentOptionId, flow) {
+        _prepareTransactionRouteParams: function (code, paymentOptionId, flow) {
```

Comment:
For later, would be nice to uniformize methods to always receive `provider_code` (or `code`), I find it strange that `_processTokenPayment` has different params name